PostHog/Classes/PHGSession.m

```diff
@@ -0,0 +1,56 @@
+//
+//  PHGSession.m
+//  PostHog
+//
+//  Created by Eric Duong on 7/22/22.
+//  Copyright © 2022 PostHog. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "PHGSession.h"
+#import "PHGPostHogUtils.h"
+
+static int const SESSION_CHANGE_THRESHOLD = 1800;
+
+@interface PHGSession ()
+
+@property (nonatomic, copy) NSString *sessionId;
+@property (nonatomic) NSTimeInterval sessionStartTimestamp;
```

Comment:
i'm calling this a `sessionLastTimestamp` here instead to align with https://github.com/PostHog/posthog-js-lite/pull/4/files#diff-30dde81fd76ff386b3bde594b35a527f3bac4438ed7ef3183ed03ab756b4c562R34, shall we consolidate to that naming?

Reply:
Works!